chapter-06/src/main/scala/chapter06/chat_schema.scala

```diff
@@ -1,24 +1,26 @@
 package chapter06
 
+import slick.driver.JdbcDriver
```

Comment:
It looks like the _object_ `JdbcDriver` has been deprecated. I don't think we use that any place. The _trait_, we which use, looks to be still relevant.